src/Sylius/Bundle/OrderBundle/Aggregator/AdjustmentsByLabelAggregator.php

```diff
@@ -23,12 +23,12 @@ class AdjustmentsByLabelAggregator implements AdjustmentsAggregatorInterface
      */
     public function aggregate(array $adjustments)
     {
-        $aggregatedAdjustments = array();
         foreach ($adjustments as $adjustment) {
-            if (!$adjustment instanceof AdjustmentInterface) {
-                throw new \InvalidArgumentException('Each adjustments array element must implement '.AdjustmentInterface::class.'.');
-            }
+            $this->assertElementIsAdjustment($adjustment);
```

Comment:
This can be done in the same loop as aggregation, we shouldn't iterate 2 times through the same array.